x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/InferTrainedModelDeploymentAction.java

```diff
@@ -291,32 +288,13 @@ public Builder setDocs(List<Map<String, Object>> docs) {
                 return this;
             }
 
-            public Builder setInferenceTimeout(TimeValue inferenceTimeout) {
-                this.timeout = inferenceTimeout;
-                return this;
-            }
-
             public Builder setUpdate(InferenceConfigUpdate update) {
                 this.update = update;
                 return this;
             }
 
-            private Builder setInferenceTimeout(String inferenceTimeout) {
-                return setInferenceTimeout(TimeValue.parseTimeValue(inferenceTimeout, TIMEOUT.getPreferredName()));
-            }
-
-            public Builder setTextInput(List<String> textInput) {
-                this.textInput = textInput;
-                return this;
-            }
-
-            public Builder setSkipQueue(boolean skipQueue) {
-                this.skipQueue = skipQueue;
-                return this;
-            }
-
             public Request build() {
-                return new Request(id, update, docs, textInput, skipQueue, timeout);
```

Comment:
Given these parameters are still needed in the Request object, I think it makes sense to keep them around for documentation/future use...

Reply:
Heh actually turns out the whole builder class is unused :) => and it's gone.